client/blocks/plugins-update-manager/schedule-form.tsx

```diff
@@ -279,7 +281,9 @@ export const ScheduleForm = ( props: Props ) => {
 							</Text>
 						) : (
 							<Text className="info-msg">
-								Plugins not listed below are managed by WordPress.com and update automatically.
+								{ translate(
+									'Plugins not listed below are managed and updated by WordPress.com for you.'
```

Comment:
```suggestion
									'Plugins not listed below are managed and updated for you by WordPress.com.'
```

Reply:
We could probably go even one step simpler:

Plugins not listed below are automatically updated by WordPress.com.